contracts/implementation/PoolCommitter.sol

```diff
@@ -282,7 +282,12 @@ contract PoolCommitter is IPoolCommitter, IPausable, Initializable {
         TotalCommitment storage totalCommit = totalPoolCommitments[appropriateUpdateIntervalId];
         UserCommitment storage userCommit = userCommitments[msg.sender][appropriateUpdateIntervalId];
 
-        userCommit.updateIntervalId = appropriateUpdateIntervalId;
+        if (userCommit.updateIntervalId == 0) {
+            userCommit.updateIntervalId = appropriateUpdateIntervalId;
+        }
+        if (totalCommit.updateIntervalId == 0) {
+            totalCommit.updateIntervalId = appropriateUpdateIntervalId;
+        }
```

Comment:
should this be != 0?

Reply:
i think this is right, this check ensures that the value on the object is only set once

Reply:
oh woops I misread.